x-pack/plugins/infra/public/pages/metrics/index.tsx

```diff
@@ -124,7 +129,18 @@ export const MetricDetail = withTheme(
                             >
                               {({ metrics, error, loading, refetch }) => {
                                 if (error) {
-                                  return <ErrorPageBody message={error} />;
+                                  if (error.graphQLErrors) {
+                                    const invalidNodeError = error.graphQLErrors.find(
```

Comment:
Given that we're actually only interested in the boolean value, how about using `Array.prototype.some()` instead of `find()`?